packages/quantic/force-app/examples/main/lwc/exampleQuanticSummary/exampleQuanticSummary.html

```diff
@@ -0,0 +1,19 @@
+<template>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}>
+  
+    <div slot="configuration">
+      <c-configurator options={options} ontryitnow={handleTryItNow}>
+      </c-configurator>
+    </div>
+
+    <c-quantic-search-interface slot="preview" engine-id={engineId}>
+      <div class="slds-grid">
+        <div><c-quantic-summary engine-id={engineId}></c-quantic-summary></div>
+        <div><c-quantic-results-per-page engine-id={engineId}></c-quantic-results-per-page></div>
```

Comment:
I would suggest to add an action button component to force the page size to a specific value, instead of having many different components in the preview section.

Reply:
What do you mean when you say "to force the page size to a specific value"?

Reply:
I mean adding an action button like "25 results per page" that would instanciate the ResultsPerPage controller and invoke `resultsPerPage.set(25)` instead of adding more components in the preview section.

The idea is that the preview section should show only the QuanticSummary component. Previewing many components at the same time might confuse the end user.

Reply:
Oh okok I read "page size" and thought you were referring to testing the responsiveness of the component to different dimensions. Yes I agree that's a good idea.